hive/agents/qnets/ac_nets.py

```diff
@@ -86,24 +99,32 @@ def __init__(
             actor_network = torch.nn.Identity()
         else:
             actor_network = actor_net(network_output_dim)
+
+        if actor_head_init_fn is None:
+            actor_head_init_fn = partial(actor_critic_init_fn, std=0.01)
+
         feature_dim = np.prod(calculate_output_dim(actor_network, network_output_dim))
         actor_head = GaussianPolicyHead if self._continuous_action else CategoricalHead
 
         self.actor = torch.nn.Sequential(
             actor_network,
             torch.nn.Flatten(),
-            actor_head(feature_dim, action_space),
+            actor_head(feature_dim, action_space).apply(actor_head_init_fn),
         )
 
         if critic_net is None:
             critic_network = torch.nn.Identity()
         else:
             critic_network = critic_net(network_output_dim)
+
+        if critic_head_init_fn is None:
+            critic_head_init_fn = partial(actor_critic_init_fn, std=1)
```

Comment:
I think it might be better to move the head initializations to the config, to make the config more explicit, rather than having this initialization hidden in the code.

Reply:
Added it to all PPO configs. I dont think we shold remove defaults though